### PR TITLE
New strings PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**
Only merge when https://github.com/DistributedCollective/Sovryn-frontend/pull/1758 be merged. 
@soulBit 

Thanks

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -1174,6 +1178,11 @@
   "closeTradingPositionHandler": {
     "title": "Fechar posição",
     "withdrawIn": "Ativo:",
+    "amountReceived": "Amount received:",
+    "marginType": "Margin Type:",
+    "positionSize": "Position Size:",
+    "minimumReceived": "Minimum Received:",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1184]:
```suggestion
    "minimumReceived": "Mínimo a receber:",
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -1174,6 +1178,11 @@
   "closeTradingPositionHandler": {
     "title": "Fechar posição",
     "withdrawIn": "Ativo:",
+    "amountReceived": "Amount received:",
+    "marginType": "Margin Type:",
+    "positionSize": "Position Size:",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1183]:
```suggestion
    "positionSize": "Tamanho da posição:",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -1174,6 +1178,11 @@
   "closeTradingPositionHandler": {
     "title": "Fechar posição",
     "withdrawIn": "Ativo:",
+    "amountReceived": "Amount received:",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1181]:
```suggestion
    "amountReceived": "Quantidade a receber:",
```

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -1377,23 +1386,32 @@
...
       "leverage": "Alavancagem:",
       "asset": "Ativo de garantia:",
-      "direction": "Direção:",
+      "direction": "Par de negociação:",
+      "positionSize": "Tamanho da posição",
       "maintananceMargin": "Margem de manutenção:",
       "liquidationPrice": "Preço estimado de liquidação:",
-      "entryPrice": "Preço aproximado de entrada:",
+      "entryPrice": "Preço de entrada da posição:",
+      "minLiq": "Preço mínimo de liquidação: ",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1412]:
```suggestion
      "minLiq": "Preço mínimo de liquidação:",
```

**Comment by creed-victor** [comment]
Merging it to #1758 feature.

**Pull request merged**